src/common/latest/helpers/request.ts

```diff
@@ -34,18 +35,18 @@ export async function get<T>(
     }
   } catch (error) {
     if (error.response) {
-      tl.debug(`[SQ] API GET '${path}' failed, status code was: ${error.response.status}`);
+      log(LogLevel.DEBUG, `API GET '${path}' failed, status code was: ${error.response.status}`);
     } else {
-      tl.debug(`[SQ] API GET '${path}' failed, error is ${error.message}`);
+      log(LogLevel.DEBUG, `API GET '${path}' failed, error is ${error.message}`);
     }
-    throw new Error(`[SQ] API GET '${path}' failed, error is ${error.message}`);
+    throw new Error(`API GET '${path}' failed, error is ${error.message}`);
   } finally {
     clearTimeout(timeout);
   }
 }
 
 export async function getServerVersion(endpoint: Endpoint): Promise<semver.SemVer> {
   const serverVersion = await get<string>(endpoint, "/api/server/version", false);
-  tl.debug(`[SQ] Server version: ${serverVersion}`);
+  log(LogLevel.INFO, `Server version: ${serverVersion}`);
```

Comment:
Changed from `DEBUG` to `INFO`

Reply:
I think I still see some references to `[SQ]` prefix,  we could clean it up across the board
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=100618&view=logs&j=f1338668-1e5b-508a-0652-45679475677e&t=41df0c21-6a86-5c69-c5a6-7f399df43a84&l=1742

Reply:
oh ignore me, I see that the test runs SC v1 and we only updated latest, which makes sense